src/generators/non_gradient_based/growing_spheres/growing_spheres.jl

```diff
@@ -38,7 +38,7 @@ function growing_spheres_generation!(ce::AbstractCounterfactualExplanation)
     factual = ce.x
     counterfactual_data = ce.data
     target = [ce.target]
-    max_iter = 1000
+    max_iter = ce.max_iter
```

Comment:
At least with the current struct, we need `ce.convergence[:max_iter]` here (tests currently failing)

Reply:
👌